contracts/ICS20Transfer.sol

```diff
@@ -256,6 +260,18 @@ contract ICS20Transfer is
         );
     }
 
+    /// @inheritdoc IICS20Transfer
+    function insertCustomERC20(string calldata denom, address token) external onlyRole(ERC20_CUSTOMIZER_ROLE) {
+        ICS20TransferStorage storage $ = _getICS20TransferStorage();
+        require(address($._ibcERC20Contracts[denom]) == address(0), IICS20Errors.ICS20DenomAlreadyDeployed(denom));
```

Comment:
Nit: Since this could also happen if a custom ERC20 has already been set here, this could maybe just be ICS20DenomAlreadyExists?